src/pages/CRLP_RollupSetup.page

```diff
@@ -34,8 +34,15 @@
   @group-content ../../ApexDocContent/Rollups2.htm
   @description Wrapper page for the Rollups UI Lightning Container.
 -->
-<apex:page standardStylesheets="false" applyBodyTag="false" id="CRLP_Setup" showHeader="true" sideBar="false"
-           controller="CRLP_RollupSetup_CTRL" docType="html-5.0">
+<apex:page 
```

Comment:
My only question here is whether we want to split out this one line into multiple lines. I'm guessing this is auto-formatted from VSCode.

Reply:
@force2b I think it is already split out into multiple lines.  Are you not seeing that formatting?  In the files changed it has lines 38-45 for the parameters of the apex:page

Reply:
It is now, but it was a single line originally.

Reply:
Oh I get the question now.  Do we have a preference for these style changes?  I've seen some pages with it split up and others on a single line.

Reply:
Probably true. I guess I just have a dislike for lots of small lines. We can leave it though.

Reply:
Is anything different about how this gets rendered in the UI? I created a scratch org and I don't see any difference. If the change is just that we're using a $Label variable instead of hard-coded, then I don't think I have much to review.

Reply:
@sam-knox 
The change is in the text of the browser tab.  It used to say CRLP_Setup in your browser tab and now it says Customizable Rollups

![screen shot 2019-01-03 at 3 18 39 pm](https://user-images.githubusercontent.com/45406286/50667535-9b6de300-0f6e-11e9-9151-e51659d97095.png)

vs

![crlp_setup](https://user-images.githubusercontent.com/45406286/50667572-cbb58180-0f6e-11e9-9e9a-16595f56a8e5.png)

Reply:
Oh, THAT title! I didn't think to look at the browser tab. Gotcha. :-)